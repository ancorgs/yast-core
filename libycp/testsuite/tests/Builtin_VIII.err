Parsed:
----------------------------------------------------------------------
(lookup ($[1:"a", 2:"bc"], 371, "take this") == "take this")
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(size ($[]) == 0)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(size ($["key":`value]) == 1)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(foreach (`k, `v, $[1:1, 2:4, 3:9], ``{
  y2debug ("v = %1", v);
  return v;
}
) == 9)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(foreach (`k, `v, $[1:1, 2:4, 3:9], ``{
  y2debug ("v = %1", v);
  return v;
}
) == 9)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(sort ([2, 1, true, 1]) == [true, 1, 1, 2])
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(tofloat (4) == 4.00000000000000)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(tofloat (-4) == -4.00000000000000)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(tofloat (4711.00000000000) == 4711.00000000000)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(tofloat (-4711.00000000000) == -4711.00000000000)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(tofloat ("42") == 42.0000000000000)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(tofloat ("+42") == 42.0000000000000)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(tofloat ("-42") == -42.0000000000000)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(tofloat ("0xb") == 11.0000000000000)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(tofloat ("-010") == -8.00000000000000)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(tofloat ("Text") == 0.00000000000000)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(tostring (0.123450000000000, 4) == "0.1235")
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(tohexstring (31) == "0x1f")
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
toterm ("mary")
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(filterchars ("aÖBÄc", "abcdefghijklmnopqrstuvwxyz") == "ac")
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(eval (``((1 + 2))) == 3)
----------------------------------------------------------------------
