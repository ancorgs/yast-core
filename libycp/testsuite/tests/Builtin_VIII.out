(true)
(true)
(true)
(true)
(true)
(true)
(true)
(true)
(true)
(true)
(true)
(true)
(true)
(true)
(true)
(true)
(true)
(true)
(`mary ())
(true)
(true)
